src/main/java/com/gachigang/ontherun/common/exception/NotFoundException.java

```diff
@@ -1,10 +1,14 @@
 package com.gachigang.ontherun.common.exception;
 
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
 
 /**
  * Exception class representing HTTP 404(Not found).
  */
+@Component
+@PropertySource("classpath:message.properties")
```

Comment:
Hmmmm, IMHO: We need to ovveride property source bean and set this path only once as a defoult one instead of copy-pasting it. Or you can create an parent error for the whole project like 'ApplicationError' then extend it from Runtime and other exceptions extend from ApplicationError class, so you will be able to put this annonation only above of ApplicationError  class (It will work if PropertySouce is extendable annotation)

Reply:
Fixed